pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
Sorry, I know the pressure is high. I fixed the deadlock; see scijava/scijava-ui-swing#28. I released scijava-ui-swing version 0.9.3 which should let you move forward without needing imagej-legacy on your classpath. Let me know how it works for you.